test/ClangImporter/cxx_interop_ir.swift

```diff
@@ -12,3 +12,10 @@ func indirectUsage() {
 // CHECK: define hidden swiftcc void @"$s6cxx_ir24namespaceManglesIntoName3argySo2nsV1TV_tF"
 func namespaceManglesIntoName(arg: namespacedT) {
 }
+
+// CHECK-LABEL: define hidden swiftcc void @"$s6cxx_ir14accessNSMemberyyF"()
+// CHECK: %0 = call %"class.ns::T"* @{{.+doMakeT.+}}()
```

Comment:
Hm, this time it might actually be worth checking that it got the right "ns::doMakeT", which means checking both platforms' manglings.